rpa-manager/modules/ROOT/pages/processautomation-prepare-project.adoc

```diff
@@ -0,0 +1,28 @@
+= Preparing Projects
+
+A project defines the framework conditions for the automation of a process and is managed by a team. 
```

Comment:
can you say "process automation" instead of "automation of a process"?

Reply:
I feel that the term process automation is to general here, but I might be wrong.

Reply:
OK. (You're the SME!)